src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data;
+
+import java.util.ArrayList;
+
```

Comment:
Please include a header comment to indicate what this interface/class/method does?